Cargo.toml

```diff
@@ -61,6 +61,9 @@ zenoh = "1.0.1"
 ringbuffer = "0.15.0"
 strum = "0.26"
 strum_macros = "0.26"
+bitflags = { version = "2.9.0", features = ["serde"] }
+semver = { version = "1.0.26", features = ["serde"] }
+cached = "0.55.1"
```

Comment:
could you break this into a new commit `cargo: add cached, bitflags, semver crates` or something similar?